fairing/preprocessors/base.py

```diff
@@ -84,9 +84,13 @@ def context_map(self):
         return c_map
 
     def context_tar_gz(self, output_file=constants.DEFAULT_CONTEXT_FILENAME):
+        logging.info("Creating docker context: %s", output_file)
         self.input_files = self.preprocess()
+        logging.info("Adding files to context: %s", self.input_files)
```

Comment:
`context_map` files are the full set of files added to the context

Reply:
Done.